internal/suite/suite_test.go

```diff
@@ -186,7 +186,7 @@ var _ = Describe("Suite", func() {
 
 		Context("when provided with a filter", func() {
 			BeforeEach(func() {
-				focusString = `inner|\d`
```

Comment:
This is a breaking change. Users who relied on `-ginkgo.skip` overwriting the old value will be surprised... I certainly was and it took me a while to figure out why some tests no longer ran.

The reason was this default:
```
	// Skip slow or distruptive tests by default.
	flag.Set("ginkgo.skip", `\[Slow|Disruptive\]`)
	flag.Parse()
```

We overwrite that default in our Makefile, but because of this change, the skip string above remained in effect.

Reply:
+1
maybe need keep this test, keep no breaking change.